test/checkov_test.exs

```diff
@@ -62,5 +62,14 @@ defmodule CheckovTest do
     ]
   end
 
+  data_test "#{name}" do
+    assert true
+
+    where [
+      [:name],
+      ["no unused variable warning"]
+    ]
+  end
```

Comment:
I'm not in love with this test, but I couldn't figure out a better way to capture the issue. This test will always pass, obviously, but it will cause the compiler warning if someone were to regress the functionality. 

Hopefully someone will catch it. 🤷‍♂